nuxt.config.ts

```diff
@@ -131,7 +124,8 @@ const config: NuxtConfig = {
     ['@nuxt/typescript-build', {
       typeCheck: true,
       ignoreNotFoundWarnings: true
-    }]
+    }],
+    '@nuxtjs/style-resources',
```

Comment:
Per offline conversations, we move the `@nuxtjs/stype-resources` into `buildModules` to follow the [instructions on the setup](https://github.com/nuxt-community/style-resources-module/#setup)

Reply:
@delapuente FYI